detectable/src/test/java/com/synopsys/integration/detectable/detectables/bitbake/unit/GraphParserTransformerTest.java

```diff
@@ -20,22 +23,25 @@ public void parsedVersionFromLabel() {
         HashMap<String, GraphEdge> edges = new HashMap<>();
         HashMap<String, GraphNode> nodes = new HashMap<>();
 
-        addNode("name", "name\\n:version\\n/some/path/to.bb", nodes, edges);
-        BitbakeGraph bitbakeGraph = buildGraph(nodes, edges);
+        addNode("name", "name\\n:version\\n/some/meta/path/to.bb", nodes, edges);
+        Set<String> knownLayers = new HashSet<>(Arrays.asList("aaa", "meta", "bbb"));
+        BitbakeGraph bitbakeGraph = buildGraph(nodes, edges, knownLayers);
```

Comment:
Seems a bit odd BitBakeGraph needs to be tested? It looks like a model object. Perhaps the logic this is testing is better suited in a transformer or a parser. (Not necessarily related to your PR I know, but I wanted to mention it).

Reply:
I agree, we throw now as diagnostics are the only way we will get feedback. Phone home woes 😞

Reply:
I think the main objective of this test is to test is to test the transformer, but BitbakeGraph is a mix of code and data, and I think you might be pointing out that that should be separated. I'm adding a note to the bitbake audit ticket to address that.